### PR TITLE
"Towards the first adversarially robust neural network model on MNIST"

### PR DESCRIPTION
**Pull request by Dref360**

### CODE REVIEW

File: deep-learning/_posts/2018-09-27-mnist-adv.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/pdf/1805.09190v3.pdf"
+---
+
+Adversarial attacks received a great amount of attention in the past year. Despite this, all methods can still be easily fooled by most attacks, even on MNIST.
+The authors propose a new method which is less accurate, but more robust.
+
+They argue that current methods are fooled because we don't validate the prediction from the network. To do so, they compare the input image to a generated image.
+
+To generate those images, they train a VAE for each class. Using gradient descent, they can optimize in the latent space to generate the closest sample to the input image. (See fig. 1)
```
Comment (lemairecarl) [inline deep-learning/_posts/2018-09-27-mnist-adv.md]: Can you explain us the training phase and the inference phase separately, please? Is there gradient descent inside the inference phase?
